public/assets/css/welcome.css

```diff
@@ -0,0 +1,93 @@
+.load{
+    animation: slide 2s;
+}
+.slider{
+    margin:0;
+    padding: 0;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    width: 100%;
+    height: 100vh;
+    animation: slide 30s infinite;
+}
+.content{
+    color: #fff;
+    width: 100%;
+    height: 100vh;
+    background-color: rgba(0, 0, 0, 0.5);
+}
+.principal{
+    --letter-spacing: 5px;
+    font-family: var(--title-font);
+    --light: #FFFFFF;
+    left: 50%;
```

Comment:
many magic number and attention you have 2 pr in one.

Reply:
comment applied